src/src/tx_main.cpp

```diff
@@ -656,12 +655,18 @@ void ICACHE_RAM_ATTR timerCallback()
     // Indicate no telemetry packet received to the DP system
     DynamicPower_TelemetryUpdate(DYNPOWER_UPDATE_MISSED);
   }
-  TelemetryRcvPhase = ttrpTransmitting;
 
 #if defined(Regulatory_Domain_EU_CE_2400)
-    BeginClearChannelAssessment(); // Get RSSI reading here, used also for next TX if in receiveMode.
+  // The last period was receiving tlm, but nothing was received.
+  // No need to enter rxmode again, lets just read the latest GetRssiInst().
```

Comment:
So we either didnt receive a packet due to bad CRC and noise.  Or no IRQ was trigger at all, and in this case the radio will continue to sample symbols until timeout.

I did observe tlm not being received, and the following rssi instantaneous being lower than the threshold.  So this method does not mean that after every failed tlm receipt the following tx packet will not be sent.